pkg/hubble/parser/getters/getters.go

```diff
@@ -76,3 +77,10 @@ type LinkGetter interface {
 	// containing the ifindex if the link name cannot be determined.
 	Name(ifIndex uint32) string
 }
+
+// PodMetadataGetter ...
+type PodMetadataGetter interface {
```

Comment:
The term "Parent Pod" is a bit confusing; a Container is part of a Pod by default. ParentPod somehow implies the ancestor of another pod. Would it make sense to rename this?